### ISSUE TITLE
Scope attribute should be used correctly on tables

### ISSUE THREAD

**Issue by christinach** [pulibrary/static-tables#110]
### Steps to recreate
Go to https://static-tables-staging.princeton.edu/
Open the dev console.
Scan the page using axe-core tools browser extension

### URL
https://static-tables-staging.princeton.edu/

### Issues
Ensures the scope attribute is used correctly on tables

Element location: `tr:nth-child(1) > td:nth-child(1)`
`<td scope="row">1-Feb-24</td>`

### Severity
High

### WCAG Success Criterion
[SC 1.3.1](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships)

### WCAG Conformance level (A, AA, AAA)
A

### User impact
The scope attribute makes table navigation much easier for screen reader users, provided that it is used correctly. Incorrectly used, scope can make table navigation much harder and less efficient.

A screen reader operates under the assumption that a table has a header and that this header specifies a scope. Because of the way screen readers function, having an accurate header makes viewing a table far more accessible and more efficient for people who use the device.

### Recommended fix
In HTML 5, scope attributes may only be used on table header elements (<th>)

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes

[Using the scope attribute to associate header cells with data cells in data tables](https://www.w3.org/WAI/WCAG22/Techniques/html/H63.html)

[Accessibility audit_05152024_Static-Tables](https://docs.google.com/spreadsheets/d/1_-Khj5TezyEu2gVb2jJczWCUt04bt5_P8-zL0f269kQ/edit#gid=0)
